src/main/java/io/confluent/kafkarest/simpleconsumerspool/NaiveSimpleConsumerPool.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package io.confluent.kafkarest.simpleconsumerspool;
```

Comment:
Hmm, this package name seems a bit long and hard to parse. Given some suggestions below, I think we may be able to drop it, but if nothing else I'd just rename it io.confluent.kafkarest.simpleconsumer and maybe even group other classes under there.